### ISSUE TITLE
Error when running installation steps on macOS

### ISSUE THREAD

**Issue by jweill-aws** [jupyterlab-contrib/jlab-enhanced-cell-toolbar#43]
## Description

The README file recommends building the code as follows:

```
# Clone the repo to your local environment
# Change directory to the jlab_enhanced_cell_toolbar [sic] directory
# Install package in development mode
pip install -e .
# Link your development version of the extension with JupyterLab
jupyter labextension develop . --overwrite
# Rebuild extension Typescript source after making changes
jlpm run build
```

I get an error when running the second command above on macOS 11.6.2 (Big Sur) with conda 4.11.0 and Python 3.9.7.

## Reproduce

1. Run `pip3 install -e .` (Not `pip`, which runs Python 2 by default on macOS Big Sur). Succeeds.
2. Run `jupyter labextension develop . --overwrite` from the root of my local package repo.

Error occurs:

```
(base) jweill@f0189814e6a5 jlab-enhanced-cell-toolbar % jupyter labextension develop . --overwrite
Traceback (most recent call last):
  File "/Users/jweill/git/jlab-enhanced-cell-toolbar/setup.py", line 8, in <module>
    from jupyter_packaging import (
ModuleNotFoundError: No module named 'jupyter_packaging'
An error occurred.
FileNotFoundError: The Python package `.` is not a valid package, it is missing the `setup.py` file.
See the log file for details:  /var/folders/ll/xhm_lj355zng4kbnktyz3pdh0000gs/T/jupyterlab-debug-r1ig1e2x.log
```

(Will add the contents of the log in the first comment)

## Expected behavior

Command completes without errors.

## Context

macOS 11.6.2 (Big Sur) with conda 4.11.0 and Python 3.9.7.

**Comment by jweill-aws** [comment]
Full error log:
[jupyterlab-debug-r1ig1e2x.log](https://github.com/jupyterlab-contrib/jlab-enhanced-cell-toolbar/files/7938215/jupyterlab-debug-r1ig1e2x.log)

**Comment by fcollonval** [comment]
Hey @jweill-aws thanks for reporting. If you have the time to update this project to the latest extension cookiecutter, it should fix the error.

**Comment by fcollonval** [comment]
Closing as fixed by #44
